django/thunderstore/modpacks/api/experimental/views/legacyprofile.py

```diff
@@ -46,6 +48,10 @@ def post(self, request, *args, **kwargs):
             content=self.request.data["file"]
         )
         serializer = LegacyProfileCreateResponseSerializer({"key": profile.id})
+        try:
+            create_legacy_profile_metadata(profile=profile) # type: ignore
```

Comment:
This isn't now running as a celery task, which it should. Also don't use type ignore, fix the type hint instead.